lib/modules/platform/github/index.ts

```diff
@@ -191,20 +191,23 @@ export async function getRepos(): Promise<string[]> {
   try {
     if (platformConfig.isGHApp) {
       const res = await githubApi.getJson<{
-        repositories: { full_name: string }[];
+        repositories: { full_name: string; archived: boolean }[];
```

Comment:
```suggestion
        repositories: { full_name: string; archived: boolean }[];
```

please create an interface for the response. maybe it should be `GhRestRepo` ?